netci.groovy

```diff
@@ -863,7 +1042,10 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
             break
         case 'x86ryujit':
         case 'x86lb':
-            assert (scenario == 'default' || scenario == 'r2r' || scenario == 'pri1r2r' || scenario == 'gcstress15_pri1r2r' || scenario == 'longgc' || scenario == 'gcsimulator')
+            assert (scenario == 'default' || scenario == 'r2r' || scenario == 'pri1r2r' || scenario == 'gcstress15_pri1r2r' || scenario == 'longgc' || scenario == 'gcsimulator' ||
+                    scenario == 'r2r_jitstress1'  || scenario == 'r2r_jitstress2'  || scenario == 'r2r_jitstressregs1'  || scenario == 'r2r_jitstressregs2'  || 
+                    scenario == 'r2r_jitstressregs3'  || scenario == 'r2r_jitstressregs4'  || scenario == 'r2r_jitstressregs8'  || scenario == 'r2r_jitstressregsx10'  ||
+                    scenario == 'r2r_jitstressregsx80'  || scenario == 'r2r_jitminopts'  || scenario == 'r2r_jitforcerelocs')
```

Comment:
It might be good to define a constant list with all of the new R2R JIT modes - that way you can check for inclusion in that last instead of the last big section of this assert block (as well as a few other places below)